contrib/utilities/indent_common.sh

```diff
@@ -177,3 +177,21 @@ process()
   esac
 }
 
+#
+# Variant of above function that only processes files that have changed
+# since the last merge commit to master. For this, we collect all files
+# that
+#  - are new
+#  - have changed since the last merge commit to master
+#
+
+process_changed()
+{
+  LAST_MERGE_COMMIT="$(git log --format="%H" --merges --max-count=1 master)"
+
+  ( git ls-files --others --exclude-standard -- ${1};
```

Comment:
Do we really want to indent untracked files as well?
In my opinion, the `git diff` command below should be sufficient.
I just confirmed that a staged file is picked up as well.

Reply:
Absolutely. Without that it is only half as useful. (I for example end up running indent scripts before I compose a commit. So files are usually not added to the staging area.)

Reply:
OK. I don't really have a preference. Sometimes I have files that are not intended for that exact commit in the source directories. It certainly doesn't hurt if they are taken care of as well.

When reading `[...](or been added and not committed)`, I thought you meant staged files. Maybe, you want to clarify this.

Reply:
Thanks! :smiley:

Reply:
I agree that we specifically do want to indent unstaged files. These are candidates for staging and committing next.

The regular indentation script also indents these files, so this is consistent.